dashboard/src/main/resources/css/dashboard.css

```diff
@@ -24,6 +24,12 @@
    font-weight: bold;
  }
 
+ table.dependency-dashboard a,
+ table.dependency-dashboard a:link,
+ table.dependency-dashboard a:visited {
+   color: black;
```

Comment:
We should probably stick to browser default link colors to avoid surprising users.